scripts/webflow-api-sync.ts

```diff
@@ -0,0 +1,180 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import { WebflowClient } from 'webflow-api';
+import yaml from 'js-yaml';
+import { setTimeout } from 'node:timers/promises';
+import util from 'node:util';
+import type { CollectionItem, CollectionItemList } from 'webflow-api/api';
+
+const rateLimitSleep = 1000;
+
+const apiCollectionId = '66f2c79a523a1055b3da7339';
+const apiCategoriesId = '6751672fcfb952c066685384';
+const siteId = '63c092e946f9b71ff6874169';
+const domainIds = ['63d392e62ba46692fb9e082e', '63d392e62ba4666ba69e082d'];
+
+if (!process.env['WEBFLOW_CMS_API_TOKEN']) {
+    throw new Error('Missing WEBFLOW_CMS_API_TOKEN');
+}
+
+const webflow = new WebflowClient({ accessToken: process.env['WEBFLOW_CMS_API_TOKEN'] });
+
+const providersPath = 'packages/shared/providers.yaml';
+// eslint-disable-next-line import/no-named-as-default-member, @typescript-eslint/no-explicit-any
+const providers: any = yaml.load(await fs.readFile(providersPath, 'utf8'));
+
+const docsPath = 'docs-v2/integrations/all';
+const files = await fs.readdir(docsPath);
+
+// we only need a subset of providers based on how our docs are written
+const neededProviders: Record<string, any> = {};
+
+for (const file of files) {
+    if (file.endsWith('.mdx')) {
+        const filePath = path.join(docsPath, file);
+        const content = await fs.readFile(filePath, 'utf-8');
+        const lines = content.split('\n');
+
+        // find the integration line
+        const providerLine = lines.find((line) => line.startsWith('provider: '));
+        if (!providerLine) {
+            throw new Error(`Unable to find provider entry in ${file}`);
+        }
+
+        const provider = providerLine.split('provider: ')[1]?.trim();
+        if (!provider) {
```

Comment:
you could replace the 3 ifs by `content.match(/^provider: ([a-z-])$/)`

Reply:
Fixed in 01615c660